plugins/scaffolder-backend-module-bitbucket-cloud/src/autocomplete/autocomplete.ts

```diff
@@ -77,6 +77,23 @@ export async function handleAutocompleteRequest({
 
       return { results: result.map(title => ({ title })) };
     }
+    case 'branches': {
+      if (!context.workspace || !context.repository)
+        throw new InputError(
+          'Missing workspace and/or repository context parameter',
+        );
+
+      const result: string[] = [];
```

Comment:
Originally, I returned a string instead of an object. @benjdlambert made some changes to make it more flexible, which resulted in this mix I think. I agree it's not necessary, so I'll change it!